js/features/new_pet_found/NewPetFound.jsx

```diff
@@ -158,8 +157,15 @@ class NewPetFound extends React.Component {
   handleProvincesFilter (event) {
     this.props.setProvince(event.target.value)
   }
+
+  hasMissingValues () {
+    return this.props.autonomousComunity === ''
```

Comment:
this is not right, we need to check weather any of the fields have a missing value.